.github/workflows/compilers.yml

```diff
@@ -168,7 +168,9 @@ jobs:
 
     name: ${{ matrix.entry.name }}
     runs-on: ubuntu-latest
-    container: ghcr.io/ruby/ruby-ci-image:${{ matrix.entry.container || 'clang-14' }}
+    container: 
+      image: ghcr.io/fkorotkov/ruby-ci-image:${{ matrix.entry.container || 'clang-14' }}
+      options: --user root
```

Comment:
I don't understand why you need to set the root user to run the container.

Reply:
ruby/ruby-ci-image#2 introduces new `ci` user so `test_tmpdir.rb` can properly run tests. In case of GitHub Actions they don't respect `USER` statement from the Dockerfile definition and fail to use such images without this option.

Reply:
I confirmed the CI doesn't work without `options: --user root` with coming new containers adding `USER` syntax on [my testing branch on my forked repo](https://github.com/junaruga/ruby/commits/wip/ci-compilers-user
). It seems that regular user is `ci` is running the command in the container respecting `USER`.

Here is the result with the images: https://github.com/users/junaruga/packages/container/package/ruby-ci-image .

https://github.com/junaruga/ruby/runs/3404372100?check_suite_focus=true#step:3:17

```
Run id
uid=1000(ci) gid=1000(ci) groups=1000(ci),27(sudo)

Run mkdir build
mkdir: cannot create directory 'build': Permission denied
Error: Process completed with exit code 1.
```